src/flow/index.js

```diff
@@ -4,29 +4,31 @@ import { ReactFlowProvider } from 'react-flow-renderer'
 import DAGNode from './dag-node'
 import Layout from './layout'
 import { useWindowDimensions } from './hooks'
+import stepConfig from './step-config'
 
 export const nodeTypes = { DAGNode }
 
 
-const Flow = ({ data, config, dimension = useWindowDimensions() }) => { 
-
+const Flow = ({ data, config, dimension = useWindowDimensions(), stepConfig }) => { 
   const { width, height } = dimension 
   
   return (
     <ReactFlowProvider>
-      <Layout data={data} config={config} width={width} height={height} />
+      <Layout data={data} config={config} width={width} height={height} stepConfig={stepConfig} />
     </ReactFlowProvider>
   )
 }
 
 Flow.propTypes = {
   data: PropTypes.object,
   config: PropTypes.object,
-  dimension: PropTypes.object.isRequired,
+  dimension: PropTypes.object,
+  stepConfig: PropTypes.object,
 }
 Flow.defaultProps = {
   data: {},
   config: {},
+  stepConfig: stepConfig,
```

Comment:
this can be shorthanded (for good reasons such as avoiding runtime errors on field/key typo)

```js
{
  config: {},
  stepConfig,
}
```